vars/build.groovy

```diff
@@ -49,21 +52,27 @@ def call(Map args) {
   }
 }
 
-def createImageStream(imageStreams, namespace) {
-    imageStreams.each { imageStream ->
-        def isName = imageStream.metadata.name
-        def isFound = Utils.shWithOutput(this, "oc get is/$isName -n $namespace --ignore-not-found")
-        if (!isFound) {
-          Utils.ocApply(this, imageStream, namespace)
-        } else {
-          echo "image stream exist ${isName}"
-        }
+def createImageStream(imageStreams, ns) {
+    imageStreams.each { is ->
+      ocApply(this, is, ns)
     }
 }
 
-def buildProject(buildConfigs, namespace) {
+def buildProject(buildConfigs, ns) {
     buildConfigs.each { buildConfig ->
-        Utils.ocApply(this, buildConfig, namespace)
-        openshiftBuild(buildConfig: "${buildConfig.metadata.name}", showBuildLogs: 'true')
+      startBuild(buildConfig, ns)
```

Comment:
nitpick: the patch seem to be mixing 2 conventions 
```groovy
function arg1, arg2,... 
```
and 
```groovy
function(arg1, arg2, ...)
```
it would be nice to be consistent. The convention that I have seen being used is `function arg1, arg2, ...` and `x = function(arg1, arg2, ...)  # incase of assignment`

Lets please pick one and stick to it.